packages/hs-react-ui/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { storiesOf, addDecorator } from '@storybook/react';
+import { boolean, number } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { withDesign } from 'storybook-addon-designs';
+import { Dropdown } from './Dropdown';
+
+addDecorator(withDesign);
+const design = {
+  type: 'figma',
+  url: 'https://www.figma.com/file/3r2G00brulOwr9j7F6JF59/Generic-UI-Style?node-id=102%3A28',
+};
+
+storiesOf('Dropdown', module).add(
+  'Basic',
+  () => {
+    const options = ['Apple', 'Banana', 'Grapefruit', 'Grape', 'Orange', 'Watermelon'];
+    return (
+      <>
+        <label htmlFor="fave-fruit">What's your favorite fruit?</label>
+        <Dropdown
```

Comment:
one thing that concerns me is that the container should be the source of truth

Reply:
Can you elaborate on that?